packages/app/android/gradle.properties

```diff
@@ -1,3 +1,4 @@
-org.gradle.jvmargs=-Xmx1536M
+org.gradle.jvmargs=-Xmx4G
 android.useAndroidX=true
 android.enableJetifier=true
+kotlin.jvm.target.validation.mode = IGNORE
```

Comment:
Can you please explain this to me?
Looks like it is needed when the java and Kotlin jvm version mismatch? Can't we avoid this by specifying the correct versions?

Reply:
I tried specifying the correct versions but I think this problem is not in our code but in the libraries that are included. I couldn't find a way to make it work other than with this line.